curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-through-an-array-with-a-for-loop.english.md

```diff
@@ -16,7 +16,7 @@ for (var i = 0; i < arr.length; i++) {
 }
 ```
 
-Remember that Arrays have zero-based numbering, which means the last index of the array is length - 1. Our <dfn>condition</dfn> for this loop is <code>i < arr.length</code>, which stops when <code>i</code> is at length - 1.
+Remember that Arrays have zero-based numbering, which means the last index of the array is <code>length - 1</code>. Our <dfn>condition</dfn> for this loop is <code>i < arr.length</code>, which stops when <code> i </code> is at <code>length - 1</code>.
```

Comment:
While we are fixing the wording for this challenge, let's change `numbering` to `indexing` and also remove the spaces inside the one code tag.
```suggestion
Remember that Arrays have zero-based indexing, which means the last index of the array is <code>length - 1</code>. Our <dfn>condition</dfn> for this loop is <code>i < arr.length</code>, which stops when <code>i</code> is at <code>length - 1</code>.
```